projects/rero/ng-core/src/lib/record/record.service.ts

```diff
@@ -359,34 +359,26 @@ export class RecordService {
    * @return throwError
    */
   private _handleError(error: HttpErrorResponse): Observable<Error> {
```

Comment:
The return value is not an observable

Reply:
`throwError` return an Observable of Error. This is right ;-)